doc/overview.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Introduction
+author: Frederik Rietdijk
+date: 2015-11-25
+---
+
+The Nix Packages collection (Nixpkgs) is a set of over 6,500 packages for the
```

Comment:
That number is probably more like over 30,000. Nix-env lists 13,406 builds, and there are (at least) another 12,000 packages for Haskell and 7,000 for R in the database that nix-env won't see.